agent/internal/config/secret.go

```diff
@@ -8,18 +8,19 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"syscall"
 
 	"github.com/ProtonMail/gopenpgp/v2/crypto"
 )
 
 type ConfigFromFile string
 
-func (field *ConfigFromFile) SetValue(location string) error {
-	if location == "" {
+func (field *ConfigFromFile) SetValue(path string) error {
```

Comment:
Using `path` as variable name would conflict/shadow the `path` package from stdlib, it is better be avoided, it is painful when it happens.

Reply:
Fixed.